js/src/flowAsState.ts

```diff
@@ -41,7 +41,9 @@ function flowAsState<T>(next: NextX<T>, ...args: any[]) {
     }
   };
 
-  loop();
+  loop().catch((error) => {
+    console.error(error);
```

Comment:
add context information to the logged error, so it can be identified as coming from the flowAsState function

Reply:
done